bpf-gpl/conntrack.h

```diff
@@ -504,17 +477,7 @@ static CALI_BPF_INLINE struct calico_ct_result calico_ct_v4_lookup(struct cali_t
 		k = ct_make_key(srcLTDest, ct_ctx->proto, ct_ctx->src, ct_ctx->dst, ct_ctx->sport, ct_ctx->dport);
 		v = cali_v4_ct_lookup_elem(&k);
 		if (!v) {
-			if (CALI_F_FROM_HOST &&
```

Comment:
If we settle on this part - which is part of a different PR, the rest LGTM, nice :+1: